doc/source/dtypes.rst

```diff
@@ -0,0 +1,121 @@
+.. currentmodule:: geopandas
+
+.. ipython:: python
+   :suppress:
+
+   import geopandas as gpd
+   world = gpd.GeoDataFrame().from_file('source/_example_data/naturalearth_lowres.shp')
+   world = world.rename(columns={'geometry': 'borders'}).set_geometry('borders')
+
+Data Structures
+=========================================
+
+GeoPandas implements two main data structures, a ``GeoSeries`` and a
+``GeoDataFrame``.  These are subclasses of pandas ``Series`` and
+``DataFrame``, respectively.
+
+GeoSeries
+---------
+
+A ``GeoSeries`` is essentially a vector where each entry in the vector 
+is a set of shapes corresponding to one observation. An entry may consist
+of only one shape (like a single polygon) or multiple shapes that are 
+meant to be thought of as one observation (like the many polygons that
+make up the State of Hawaii or a country like Indonesia). 
+
+*geopandas* has three classes of geometric objects (which are actually *shapely* objects):
```

Comment:
three *basic* classes; geometries can also be collections (`MultiPoint`, `MultiPolygon`, `GeometryCollection`, etc.)